src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+type Props = {};
+
+export class App extends React.Component<Props, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyup);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyup);
+  }
+
+  handleKeyup = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <p className="App__message">
+          {(this.state.pressedKey)
+            ? `The last pressed key is [${this.state.pressedKey}]`
+            : 'Nothing was pressed yet'}
```

Comment:
```suggestion
  render() {
    const { pressedKey } = this.state;
    return (
      <div className="App">
        <p className="App__message">
          {(pressedKey)
            ? `The last pressed key is [${pressedKey}]`
            : 'Nothing was pressed yet'}
```
Would be great to destructure the pressedKey before